### PR TITLE
[FIX] account: forbid resequencing secured journal entries

### PR DESCRIPTION
**Pull request by yelizariev**
STEPS:

* install l10n_fr/l10n_fr_pos_cert
* create journal entries
* in "Journal Entries" menu select some records and click "[Action] >
Resequence"

BEFORE: you can modify secured records

AFTER: you see error message

---

opw-2381859

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/wizard/account_resequence.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def default_get(self, fields_list):
             active_move_ids = self.env['account.move'].browse(self.env.context['active_ids'])
         if len(active_move_ids.journal_id) > 1:
             raise UserError(_('You can only resequence items from the same journal'))
+        if active_move_ids.filtered("inalterable_hash"):
+            raise UserError(_('Some or all of the entries are secured and cannot be modified'))
```
Comment (yelizariev) [inline addons/account/wizard/account_resequence.py:34]: @FlorianGilbert, could you review this please? I'm not sure when the restriction should be applied and  which check should be done

Comment (FlorianGilbert) [reply addons/account/wizard/account_resequence.py:34]: Hey, I will talk with my PO about that and I come back to you soon!

Comment (FlorianGilbert) [reply addons/account/wizard/account_resequence.py:34]: (May be we want to add this check only for l10n_fr)

**Comment by FlorianGilbert** [comment]
Hey, I discussed with @tsb-odoo  and it has been established that for the moment we don't want to block the resequense feature for hashed journals. TSB will write a message on the ticket.

**Comment by yelizariev** [comment]
ok, closing it then

File: addons/account/wizard/account_resequence.py (hunk 1 above)
Comment (omarie-ctc) [reply addons/account/wizard/account_resequence.py:34]: @yelizariev Hello, you have to add this only for l10_fr, the law in France prohibits the modification of invoice numbers.

Comment (yelizariev) [reply addons/account/wizard/account_resequence.py:34]: This feature is mainly used by l10n_fr, so it's kind of the same. But I'm ok to add it to 10n_fr only

Comment (FlorianGilbert) [reply addons/account/wizard/account_resequence.py:34]: @yelizariev @tsb-odoo doesn't want that so, before doing anything ask him please :smile:

**Pull request closed without merging**